src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -7,37 +7,59 @@
     }
   },
   "OrchardCore": {
+    "OrchardCore.Setup": {
+      "DefaultCulture": "",
+      "SupportedCultures": [ "en" ]
+    }
+    // Uncomment to configure automatic provisioning with config.    
```

Comment:
Can you remove the Setup languages? Or merge dev? We just recently removed them from here in favour of keeping the defaults in code